packages/orval/src/write-specs.ts

```diff
@@ -198,6 +199,51 @@ export const writeSpecs = async (
     }
   }
 
+  if (output.docs) {
+    try {
+      const app = await Application.bootstrapWithPlugins({
+        entryPoints: paths,
+        // Set the custom config location if it has been provided.
+        ...(typeof output.docs === 'object'
+          ? { options: output.docs.config }
+          : {}),
+        plugin: ['typedoc-plugin-markdown'],
+      });
+      // Set defaults if the have not been provided by the external config.
+      if (!app.options.isSet('readme')) {
+        app.options.setValue('readme', 'none');
+      }
+      if (!app.options.isSet('logLevel')) {
+        app.options.setValue('logLevel', 'None');
+      }
+      const project = await app.convert();
+      if (project) {
+        let out = 'docs';
+        if (app.options.isSet('out')) {
+          // Use the output location if it has been set in the external config.
+          out = app.options.getValue('out');
+        } else if (output.workspace) {
+          // Generate the docs in the workspace.
+          out = upath.join(output.workspace, 'docs');
+        } else if (output.target) {
+          const base = upath.dirname(output.target);
+          // Generate the docs along side the output target.
+          out = upath.join(base, 'docs');
+        }
```

Comment:
I thought that the default specified location should be `docs` regardless of `workspace` or `target`.

These options are not documents, but output destinations for the application's HTTP client, so their nature is different from the document we will be adding this time.
Also, by not supporting those followers, we get the simplicity of the source code.

```suggestion
        const out = app.options.getValue('out') || 'docs';
```

Reply:
Thanks. `app.options.getValue('out')` defaults to `docs` inside TypeDoc so we can simplify this slightly further and drop the ` || 'docs'`